modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -29,46 +27,29 @@ spec:
   agent:
     type: eBPF
     ebpf:
+      privileged: true <1>
+      features: <2>
+        - PacketDrop
+        - FlowRTT
       flowFilter:
-        action: Accept  <1>
-        cidr: 172.210.150.1/24 <2>
-        protocol: SCTP
-        direction: Ingress
-        destPortRange: 80-100
-        peerIP: 10.10.10.10
-        enable: true    <3>
+        enable: true <3>
+        rules:
+          - action: Accept  <4>
+            cidr: 172.210.150.1/24 <5>
+            protocol: SCTP
+            direction: Ingress
+            destPortRange: 80-100
+            peerIP: 10.10.10.10
+          - action: Accept
+            cidr: 172.30.0.0/16
+            pktDrops: true <6>
+          - action: Reject
```

Comment:
remove this Reject rule but need to mention somewhere that by default all other flows are rejected